rust/bridge/node/src/lib.rs

```diff
@@ -11,6 +11,19 @@ use std::ops::Deref;
 
 struct DefaultFinalize<T>(T);
 
+/*
+Starting in https://github.com/neon-bindings/neon/pull/601 Neon uses
+napi_get_all_property_names which was introduced in N-API 6. However
+N-API 6 was first supported in Node 12.70 while the version of
+Electron used in Desktop is only 12.30
+
+On Linux and macOS the link still succeeds and the symbol is simply
+undefined. However on Windows the DLL fails to link when this is
+missing.
+*/
+#[no_mangle]
+extern "C" fn napi_get_all_property_names() { std::process::abort(); }
```

Comment:
Neon 0.7.0 fixes this issue by allowing us to declare that we only need `napi-1` as a Cargo feature. #65 includes that change, but you could extract it here as well.